spec/plugin_mixins/jdbc/value_tracking_spec.rb

```diff
@@ -0,0 +1,18 @@
+# encoding: utf-8
+require "logstash/plugin_mixins/jdbc/value_tracking"
+
+module LogStash module PluginMixins module Jdbc
+  describe ValueTracking do
+
+    let(:yaml_date_source) { "--- !ruby/object:DateTime '2023-06-15 09:59:30.558000000 +02:00'\n" }
+
+    context "#load_yaml" do
```

Comment:
Can we have tests that verify a few different scenarios - `Time`, `DateTime`, `BigDecimal`? I also wonder if we should have a test that uses the `build_last_value_tracker(plugin)` to do `read` and `write` these values too

Reply:
Yep, however I think that testing  `build_last_value_tracker` is a little bit out of context of this PR. We can avoid to push 2 PRs, one for JRuby 9.4 updates and another to cover with test existing code, and commit everything to this

Reply:
I'm ok with that

Reply:
Done:
- with commit 743f919279f11048ceaa6865235ab8cc50a356bf `#load_yaml` has been covered for Time and BigDecimal cases
- while with 9a640a724c0dc695d0ddca6f2ef69f367d6b90ea the method `#build_last_value_tracker` has been exercised